docker-compose.test.watch.yaml

```diff
@@ -1,6 +1,6 @@
 services:
   ffc-sfd-messages-processor:
     command: >
-      sh -c "npm run test:watch"
+      sh -c "git config --global --add safe.directory /home/node && npm run test:watch"
```

Comment:
Wondering if we need to change the command back to the original (`sh -c "npm run test:watch"`) whenever we merge into main since it's only us two (Simon and I) who need this command in order to run tests?

Reply:
Yep agree

Reply:
Great, have changed it back.